kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/OrderRepository.java

```diff
@@ -160,6 +174,12 @@ public void onChanged(Payment payment) {
 		}
 	}
 
+	private void sendEarnPaymentConfirmed(Payment payment) {
+		if (payment.isSucceed() && payment.getAmount() != null && payment.isEarn()) {
```

Comment:
do you need to check payment != null?

Reply:
no, payment never null when it gets here